src/components/Button.js

```diff
@@ -2,16 +2,35 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = props => {
-  const { name } = props;
+  const {
+    name, btnClass, color, wide,
+  } = props;
   return (
-    <button type="button">
+    <button
+      type="button"
+      className={
+        btnClass
```

Comment:
- Is the `btnClass` prop really necessary :thinking: ? It seems that right now all buttons use this class, therefore, it is not needed to be declared as prop.

Reply:
Thank you. I'll try to refactor in the next milestone.